notebooks/02-ingestion/12-spatial-dimensions.ipynb

```diff
@@ -0,0 +1,709 @@
+    "  },\n",
+    "  \"metric\": {\n",
+    "    \"type\": \"dimension\",\n",
+    "    \"ordering\": {\n",
+    "      \"type\": \"lexicographic\"\n",
+    "    }\n",
+    "  },\n",
+    "  \"threshold\": 1001,\n",
+    "  \"intervals\": {\n",
+    "    \"type\": \"intervals\",\n",
+    "    \"intervals\": [\n",
+    "      \"-146136543-09-08T08:23:32.096Z/146140482-04-24T15:36:27.903Z\"\n",
+    "    ]\n",
+    "  },\n",
+    "  \"filter\": {\n",
+    "    \"type\": \"spatial\",\n",
+    "    \"dimension\": \"address_spatial\",\n",
+    "    \"bound\": {\n",
+    "        \"type\": \"radius\",\n",
+    "        \"coords\": [-10.0, 20.0],\n",
+    "        \"radius\": 10.0\n",
+    "      }\n",
+    "  },\n",
+    "  \"granularity\": {\n",
+    "    \"type\": \"all\"\n",
+    "  },\n",
+    "  \"aggregations\": [\n",
+    "    {\n",
+    "      \"type\": \"doubleMin\",\n",
+    "      \"name\": \"min_income\",\n",
+    "      \"fieldName\": \"income\"\n",
+    "    },\n",
+    "    {\n",
+    "      \"type\": \"doubleMax\",\n",
+    "      \"name\": \"max_income\",\n",
+    "      \"fieldName\": \"income\"\n",
+    "    },\n",
+    "    {\n",
+    "      \"type\": \"count\",\n",
+    "      \"name\": \"user_count\"\n",
+    "    }\n",
+    "  ],\n",
+    "  \"context\": {\n",
+    "    \"queryId\": \"2dcbd3c7-cebe-4bb9-8b8b-d1887deda9ea\",\n",
+    "    \"sqlOuterLimit\": 1001,\n",
+    "    \"sqlQueryId\": \"2dcbd3c7-cebe-4bb9-8b8b-d1887deda9ea\",\n",
+    "    \"useNativeQueryExplain\": False\n",
+    "  }\n",
+    "}\n",
+    "\n",
+    "headers = {\n",
+    "  'Content-Type': 'application/json'\n",
+    "}\n",
+    "\n",
+    "result = rest_client.post(\"/druid/v2\", json.dumps(radius_filter_query), headers=headers)\n",
+    "\n",
+    "json.loads(result.text)[0]['result']\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88ca6f49-1f53-41ce-8ae1-cb89c8f70cf8",
+   "metadata": {},
+   "source": [
+    "### Filter using a polygon area\n",
+    "The next cell is an example of a polygon bound.\n",
+    "A polygon describes a closed irregular shape where any point that falls within the irregular shape will be included in the results. The shape is defined by a list of coordinates in two arrays:\n",
+    "- \"abscissa\" is the list of horizontal coordinates\n",
+    "- \"ordinate\" is the list of vertical coordinates \n",
+    "\n",
+    "This example uses a rough outline of Africa:\n",
+    "\n",
+    "```\n",
+    "\"filter\": {\n",
+    "    \"type\": \"spatial\",\n",
+    "    \"dimension\": \"address_spatial\",\n",
+    "    \"bound\": {\n",
+    "        \"type\": \"polygon\",\n",
+    "        \"abscissa\": [30.942493, 36.917158, 22.799200, 3.914715, -0.286389, -35.494284, 11.654848, 30.942493 ],\n",
+    "        \"ordinate\": [32.434730, 9.758948 ,-16.959803,-8.524121,  7.766978,  19.844799, 51.309644, 32.434730 ]\n",
+    "      }\n",
+    "  },\n",
+    "```\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
```

Comment:
Run this cell to ...

(Add a sentence describing what this code does.)

Reply:
done